tests/network-tests/run-migration-tests.sh

```diff
@@ -4,97 +4,75 @@ set -e
-    rm ./assets/TestVideo__rejectedContent.json || true
+TARGET_RUNTIME_TAG=${TARGET_RUNTIME_TAG:=latest}
+# The joystream/node docker image tag to start the chain with
+RUNTIME_TAG=${RUNTIME_TAG:=sumer}
+# Post migration assertions by means of typescript scenarios required
+POST_MIGRATION_ASYNC_ASSERTIONS=${POST_MIGRATION_ASYNC_ASSERTIONS=$true}
+# The joystream/node docker image tag to start the chain with
+RUNTIME_TAG=${RUNTIME_TAG:=sumer}
+# RPC endpoint for live RUNTIME testnet 
+export WS_RPC_ENDPOINT="wss://testnet-rpc-3-uk.joystream.org"
+
+# source common function used for node setup
+source ./node-utils.sh
+source ./.env
+
+#######################################
+# use fork-off to generate a chainspec file with the current s
+# Globals:
+#   DATA_PATH
+# Arguments:
+#   None
+#######################################
+function fork_off_init() {
+    # chain-spec-raw already existing
+
+    if ! [[ -f ${DATA_PATH}/storage.json ]]; then
+	curl http://testnet-rpc-3-uk.joystream.org:9933 -H \
+	     "Content-type: application/json" -d \
+	     '{"jsonrpc":"2.0","id":1,"method":"state_getPairs","params":["0x"]}' \
+	     > ${DATA_PATH}/storage.json	
+    fi
+
+    if ! [[ -f ${DATA_PATH}/schema.json ]]; then
+	cp $SCRIPT_PATH/../../types/augment/all/defs.json ${DATA_PATH}/schema.json
+    fi
+
+    id=$(docker create joystream/node:${TARGET_RUNTIME_TAG})
+    docker cp $id:/joystream/runtime.compact.wasm ${DATA_PATH}/runtime.wasm
+    
+    yarn workspace api-scripts tsnode-strict src/fork-off.ts
 }
 
-function pre_migration_hook() {
-  sleep 10 # needed otherwise docker image won't be ready yet
-  # Display runtime version
-  yarn workspace api-scripts tsnode-strict src/status.ts | grep Runtime
-
-  # assume older version of joystream-cli is installed globally. So we run these commands to
-  # work against older runtime. Assert it is version  `@joystream/cli/0.5.1` ?
-  joystream-cli --version
-
-  joystream-cli account:choose --address 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY # Alice
-  echo "creating 1 channel"
-  joystream-cli content:createChannel --input=./assets/TestChannel.json --context=Member || true
-  echo "adding 1 video to the above channel"
-  joystream-cli content:createVideo -c 1 --input=./assets/TestVideo.json || true
-
-  # Confirm channel and video created successfully
-  joystream-cli content:videos 1
-  joystream-cli content:channel 1
+# entrypoint
+function main {
+    CONTAINER_ID=""
+
+    echo "**** CREATING EMPTY CHAINSPEC ****"
+    create_initial_config
+    create_chainspec_file
+    convert_chainspec
+    echo "**** EMPTY CHAINSPEC CREATED SUCCESSFULLY ****"
+
+    # use forkoff to update chainspec with the live state + update runtime code
+    fork_off_init
+
+    echo "***** STARTING NODE WITH FORKED STATE *****"
+    export JOYSTREAM_NODE_TAG=$RUNTIME_TAG
+    CONTAINER_ID=$(start_node)
+
+    # 120 sec needed to load the chainspec
+    sleep 120
```

Comment:
I propose we do the following to have the initial genesis state be written to db so we don't have to wait an arbitrary amount of time which needs to be different depending on what hardware is running:
```suggestion
   docker-compose -f ../../docker-compose.yml run -v ${DATA_PATH}:/spec joystream-node export-state --chain /spec/chain-spec-raw.json --base-path /data --pruning archive > ${DATA_PATH}/exported-state.json
   # short delay for node to startup
    sleep 30
```

The `--base-path /data` argument is important to be the same as one used in node-utils.sh to start the node with the same chain data generated in this step.